### PR TITLE
Updated chia init -c [directory] to copy CA

### PR DESCRIPTION
**Pull request by eFishCent**
Updated `chia init -c [directory]` to allow user to specify where their Farmer CA directory is.
Delete old trusted.crt and trusted.key if it exists.
Supports migrating config/ssl from beta26.

### CODE REVIEW

File: src/cmds/init.py [hunk 1]
```diff
@@ -36,12 +36,12 @@ def help_message():
     print(
         """
         chia init (migrate previous version configuration to current)
-        chia init -c (creates new TLS certificates signed by your CA)
+        chia init -c [directory] (creates new TLS certificates signed by your CA directory)
```
Comment (hoffmang9) [inline src/cmds/init.py:39]: "signed by your CA in directory"

File: src/cmds/init.py [hunk 2]
```diff
@@ -155,18 +166,6 @@ def migrate_from(
         return 0
     print(f"\n{old_root} found")
     print(f"Copying files from {old_root} to {new_root}\n")
-    copied = []
```
Comment (hoffmang9) [inline src/cmds/init.py:158]: We use this generically to copy e.g. the databases over so I think you can't remove it. What we do is just change the URIs as needed down below

Comment (hoffmang9) [reply src/cmds/init.py:158]: Maybe I'm missing something but I think this change would keep db/blockchain_v23.db from copying over?

**Review by hoffmang9 — CHANGES_REQUESTED** [review]
A couple of issues that could be me not grokking the change.

File: src/cmds/init.py (hunk 1 above)
Comment (eFishCent) [reply src/cmds/init.py:39]: Updated, will push changes after testing DB copy.

Comment (hoffmang9) [reply src/cmds/init.py:39]: That also sounds stilted - just want to not say "your CA directory" - maybe "by your CA found in directory"?

File: src/cmds/init.py (hunk 2 above)
Comment (eFishCent) [reply src/cmds/init.py:158]: Tested my refactored code, it copies the db just fine.
The `copied = []` was legacy variable that was eliminated when I refactored the code. I did entire search for this variable and it isn't used anywhere. Here's the original use: https://github.com/Chia-Network/chia-blockchain/commit/e2b3b650c6ebd78a041c36b322c49f04e1d8b5d3

**Review by eFishCent — COMMENTED** [review]
Double checked for `copied` in file, none used anywhere in entire src tree.

**Review by hoffmang9 — APPROVED** [review]
lgtm

**Comment by hoffmang9** [comment]
I may have borked your change to the CHANGELOG.md - if so, please PR a fix on that at your convenience.

**Pull request merged**